specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -4,149 +4,85 @@ This specification refers to the following
 documents. The terms used to refer to the documents in this
 specification are included in parentheses.
 
- _Java™ Platform, Enterprise Edition
-Specification Version 8_ (this specification). Available at
-_http://jcp.org/en/jsr/detail?id=366_ .
+Jakarta™ EE Platform Specification Version 8. Available at: <link needed>.
```

Comment:
Also, why did you remove all of the italics?  The underscores before and after each entry.

Reply:
Italics would be the normal bibliographic reference style.

Reply:
I can put them back, I didn't realize they were for italics. I had assumed they were some invalid chars that got copied over from the original spec document (perhaps a PDF or something)